src/Assignment4/Header/index.js

```diff
@@ -1,15 +1,20 @@
+import { Outlet } from "react-router-dom";
+
 import "./styles.css";
 import logo from "../../logo/React.webp";
 import usericon from "../../logo/usericon.png";
 import searchicon from "../../logo/searchicon.png";
 
 const Header = ({}) => (
```

Comment:
you can remove the empty object.. is it needed?

Reply:
removed